src/logistration/tests/RegistrationPage.test.jsx

```diff
@@ -120,6 +357,16 @@ describe('./RegistrationPage.js', () => {
           ...initialState.logistration.thirdPartyAuthContext,
           secondaryProviders: [secondaryProviders],
         },
+        formData: {
```

Comment:
I believe this is required in each test case, can we set this in `initialState` so that we don't need to repeat this in. each test?
https://github.com/edx/frontend-app-logistration/blob/f0fad2b4cb308e3de372738cc395c0ed7e354a25/src/logistration/tests/RegistrationPage.test.jsx#L18-L28